colaraz/lms/static/js/header/header.js

```diff
@@ -462,3 +465,73 @@ function removeNotificationsCount(){
         markNotificationsAsRead();
     }
 }
+
+
+function getAndPopulateJobAlerts() {
+    $.ajax({
+        type: "GET",
+        url: colarazJobAlertsFetchingUrl,
+        success: function (resp) {
+            let job_alerts = "";
+            if (resp.d.length) {
+                resp.d.forEach(element => {
+                    job_alerts += createJobAlert(element.Image, element.Heading, element.Message, element.RelativeTime);
+                });
+            } else {
+                job_alerts = "<li><p>No new job alerts</p></li>";
+            }
+
+            handleJobAlertsListing(job_alerts);
+            handleJobAlertsCount(resp.d.length);
+
+        },
+        error: function (resp) {
+            console.error(`Job Alerts API gave following error: ${resp.Message}`);
+        },
+    });
+
+    function handleJobAlertsListing(job_alerts) {
+        $("#job-alerts-list").html(job_alerts);
```

Comment:
Since edX has used single quotes everywhere else, I think we should too for consistency

Reply:
done.